Gemfile.lock

```diff
@@ -1,67 +1,67 @@
 GEM
   remote: https://rubygems.org/
   specs:
-    actioncable (6.0.3.2)
-      actionpack (= 6.0.3.2)
+    actioncable (6.0.3.4)
+      actionpack (= 6.0.3.4)
       nio4r (~> 2.0)
       websocket-driver (>= 0.6.1)
-    actionmailbox (6.0.3.2)
-      actionpack (= 6.0.3.2)
-      activejob (= 6.0.3.2)
-      activerecord (= 6.0.3.2)
-      activestorage (= 6.0.3.2)
-      activesupport (= 6.0.3.2)
+    actionmailbox (6.0.3.4)
+      actionpack (= 6.0.3.4)
+      activejob (= 6.0.3.4)
+      activerecord (= 6.0.3.4)
+      activestorage (= 6.0.3.4)
+      activesupport (= 6.0.3.4)
       mail (>= 2.7.1)
-    actionmailer (6.0.3.2)
-      actionpack (= 6.0.3.2)
-      actionview (= 6.0.3.2)
-      activejob (= 6.0.3.2)
+    actionmailer (6.0.3.4)
+      actionpack (= 6.0.3.4)
+      actionview (= 6.0.3.4)
+      activejob (= 6.0.3.4)
       mail (~> 2.5, >= 2.5.4)
       rails-dom-testing (~> 2.0)
-    actionpack (6.0.3.2)
-      actionview (= 6.0.3.2)
-      activesupport (= 6.0.3.2)
+    actionpack (6.0.3.4)
+      actionview (= 6.0.3.4)
+      activesupport (= 6.0.3.4)
       rack (~> 2.0, >= 2.0.8)
       rack-test (>= 0.6.3)
       rails-dom-testing (~> 2.0)
       rails-html-sanitizer (~> 1.0, >= 1.2.0)
-    actiontext (6.0.3.2)
-      actionpack (= 6.0.3.2)
-      activerecord (= 6.0.3.2)
-      activestorage (= 6.0.3.2)
-      activesupport (= 6.0.3.2)
+    actiontext (6.0.3.4)
+      actionpack (= 6.0.3.4)
+      activerecord (= 6.0.3.4)
+      activestorage (= 6.0.3.4)
+      activesupport (= 6.0.3.4)
       nokogiri (>= 1.8.5)
-    actionview (6.0.3.2)
-      activesupport (= 6.0.3.2)
+    actionview (6.0.3.4)
+      activesupport (= 6.0.3.4)
       builder (~> 3.1)
       erubi (~> 1.4)
       rails-dom-testing (~> 2.0)
       rails-html-sanitizer (~> 1.1, >= 1.2.0)
-    activejob (6.0.3.2)
-      activesupport (= 6.0.3.2)
+    activejob (6.0.3.4)
+      activesupport (= 6.0.3.4)
       globalid (>= 0.3.6)
-    activemodel (6.0.3.2)
-      activesupport (= 6.0.3.2)
-    activerecord (6.0.3.2)
-      activemodel (= 6.0.3.2)
-      activesupport (= 6.0.3.2)
-    activestorage (6.0.3.2)
-      actionpack (= 6.0.3.2)
-      activejob (= 6.0.3.2)
-      activerecord (= 6.0.3.2)
+    activemodel (6.0.3.4)
+      activesupport (= 6.0.3.4)
+    activerecord (6.0.3.4)
+      activemodel (= 6.0.3.4)
+      activesupport (= 6.0.3.4)
+    activestorage (6.0.3.4)
+      actionpack (= 6.0.3.4)
+      activejob (= 6.0.3.4)
+      activerecord (= 6.0.3.4)
       marcel (~> 0.3.1)
-    activesupport (6.0.3.2)
+    activesupport (6.0.3.4)
       concurrent-ruby (~> 1.0, >= 1.0.2)
       i18n (>= 0.7, < 2)
       minitest (~> 5.1)
       tzinfo (~> 1.1)
       zeitwerk (~> 2.2, >= 2.2.2)
     addressable (2.7.0)
       public_suffix (>= 2.0.2, < 5.0)
-    autoprefixer-rails (9.8.6.1)
+    autoprefixer-rails (10.0.1.0)
```

Comment:
### [autoprefixer-rails](https://github.com/ai/autoprefixer-rails)

**Major** version upgrade :chart_with_upwards_trend::exclamation: 9.8.6.1 → 10.0.1.0

[_[change-log](https://github.com/ai/autoprefixer-rails/blob/master/CHANGELOG.md), [source-code](https://github.com/ai/autoprefixer-rails)_]





<details>
<summary>Commits</summary>

A change of **30** commits. See the full changes on [the compare page](https://github.com/ai/autoprefixer-rails/compare/9.8.6.1...10.0.1.0).

These are the first 10 commits:
- (5c6ad43) [Fix typo in README.md](https://github.com/ai/autoprefixer-rails/commit/5c6ad43c7b091efad1cf8f248505ea45b4597cab)
- (1edbc0c) [Fix link](https://github.com/ai/autoprefixer-rails/commit/1edbc0c5fca0a4e3426883fe854cf95574e34a2e)
- (43393d8) [Print deprecation warning only once (#173)](https://github.com/ai/autoprefixer-rails/commit/43393d87275627e44f282915e0c61fffdfa8dc03)
- (85ca064) [Release 9.8.6.2 version](https://github.com/ai/autoprefixer-rails/commit/85ca06418f717533b456d16879291c94bc882282)
- (597950e) [Move show_deprecation_message! to Processor (#175)](https://github.com/ai/autoprefixer-rails/commit/597950eb75ff57a0f0ebf8facb903a990fd238af)
- (6800e4f) [Release 9.8.6.3 version](https://github.com/ai/autoprefixer-rails/commit/6800e4f88d5aade8886f6edc2be437d57101d58b)
- (f1042be) [build: use rollup to build vendor/autoprefixer.js (#176)](https://github.com/ai/autoprefixer-rails/commit/f1042beab8f697f00b4851089d217bda89394121)
- (9306d3e) [Remove deprecation warnings. (#177)](https://github.com/ai/autoprefixer-rails/commit/9306d3ed5ad034dff1d293ce3b73d3f9df85af70)
- (b4d81b9) [Release 9.8.6.4 version](https://github.com/ai/autoprefixer-rails/commit/b4d81b9bc6184d9459ed95ef6ff9552065e57f88)
- (9cff907) [Revert autoprefixer.js](https://github.com/ai/autoprefixer-rails/commit/9cff9076dea701dd43126be7f0395b9e6cb45802)


</details>